apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -1,43 +1,109 @@
 import {expect} from '../../../util/configuredChai';
 import React from 'react';
 import sinon from 'sinon';
+import {replaceOnWindow, restoreOnWindow} from '../../../util/testUtils';
 let project = require('@cdo/apps/code-studio/initApp/project');
 
-describe('project.getProjectUrl', function () {
+describe('project.js', () => {
```

Comment:
Added a top-level describe to this file so many of the changes are whitespace-only.